tools/ormolu.sh

```diff
@@ -72,14 +72,26 @@ echo "language extensions: $LANGUAGE_EXTS"
 
 FAILURES=0
 
+if [ -t 1 ]; then
```

Comment:
why this trivial conditional?

anyway i like the changes!

Reply:
`[ -t 1 ]` returns `0` if the standard output is a `tty`. This check is to prevent the script from printing escape sequences when it runs in CI, or any other environment which is not a shell, basically.

Reply:
Ah, my bad: I read `-t` as `isTrue`.  Checking the man page would have helped.  :)